src/components/TipInput.vue

```diff
@@ -125,22 +112,22 @@ export default {
     };
   },
   computed: {
-    ...mapGetters(['account', 'balance', 'isLoggedIn', 'loading']),
+    ...mapGetters(['account', 'loading']),
     eventPayload() {
       return `${this.tip.id}:${this.show}`;
     },
     deepLink() {
-      const url = new URL(`${process.env.VUE_APP_WALLET_URL}/tip`);
+      const url = new URL(`${process.env.VUE_APP_WALLET_URL}/${this.isRetip ? 're' : ''}tip`);
       url.searchParams.set('id', this.tip.id);
       url.searchParams.set('x-success', window.location);
       url.searchParams.set('x-cancel', window.location);
       return url;
     },
     isSendMessageDataValid() {
-      return this.message.trim().length > 0 && !this.value;
```

Comment:
there was some logic behind this (tough not very nice implemented):
You either make a tip + on-chain comment (needs tip value >0 and a message) or make an off-chain comment (where tip value is 0)

Reply:
> there was some logic behind this (tough not very nice implemented):
> You either make a tip + on-chain comment (needs tip value >0 and a message) or make an off-chain comment (where tip value is 0).
> Third option is "retip" where you don't really add a message AFAIK
> May be double check that (@mradkov)?

Yes, correct.